documentation/docs/09-link-options.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Link options
+---
+
+In SvelteKit, `<a>` elements (rather than framework-specific `<Link>` components) are used to navigate between the routes of your app. If the user clicks on a link whose `href` is 'owned' by the app (as opposed to, say, a link to an external site) then SvelteKit will navigate to the new page by importing its code and then calling any `load` functions it needs to fetch data.
+
+You can customise the behaviour of links with `data-sveltekit-*` attributes. These can be applied to the `<a>` itself, or to a parent element.
+
+### data-sveltekit-prefetch
+
+To get a head start on importing the code and fetching the page's data, use the `data-sveltekit-prefetch` attribute, which will start loading everything as soon as the user hovers over the link (on a desktop) or touches it (on mobile), rather than waiting for the `click` event to trigger navigation. Typically, this buys us an extra couple of hundred milliseconds, which is the difference between a user interface that feels laggy, and one that feels snappy.
+
+To apply this behaviour across the board, add the attribute to a parent element (or even the `<body>` in your `src/app.html`):
+
+```html
+/// file: src/routes/+layout.svelte
+<main data-sveltekit-prefetch>
+	<slot />
+</main>
+```
+
+> You can also programmatically invoke `prefetch` from `$app/navigation`.
+
+### data-sveltekit-reload
+
+Occasionally, we need to tell SvelteKit not to handle a link, but allow the browser to handle it. Adding a `data-sveltekit-reload` attribute to a link...
+
+```html
+<a data-sveltekit-reload href="/path">Path</a>
+```
+
+...will cause a full-page navigation when the link is clicked.
+
+Links with a `rel="external"` attribute will receive the same treatment. In addition, they will be ignored during [prerendering](/docs/page-options#prerender).
+
```

Comment:
I know you say above that all of the `data-sveltekit-` attributes can also be specified on parent elements, but is it weird to only explicitly call it out for `-prefetch` and not `-reload` or `-noscroll`?

Reply:
:+1: That's fair.